test/test_page_without_a_file.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require "helper"
+
+class TestPageWithoutAFile < JekyllUnitTest
+  def setup_page(*args)
```

Comment:
tiny thing: Instead of overriding this entire method to have the only difference in having `PageWithoutAFile.new` instead of `Page.new` ,what do you think of changing the method signature to something like:
```ruby
def setup_page(*args, klass: Page)
```

Reply:
Thank you for the suggestion. I had considered using a similar signature (`klass=Page, *args`), but opted out since I'm creating a `Jekyll::Page` instance only once in this new unit-test-class.

That said, I like your syntax with Keyword Arguments. Improves future tests-scalability. :+1: